dts/arm/atmel/samx7x.dtsi

```diff
@@ -472,6 +472,15 @@
 			clocks = <&pmc PMC_TYPE_PERIPHERAL 46>;
 			status = "disabled";
 		};
+
+		smc: smc@40080000 {
```

Comment:
Move this block ordered by the HW address 0x40080000, it is possible before the I2C0 I think.